internal/pkg/service/buffer/task/orchestrator/orchestrator.go

```diff
@@ -96,8 +95,6 @@ type Source[T any] struct {
 	// WatchPrefix defines an etcd prefix that is watched by GetAllAndWatch.
 	// Each event triggers new task.
 	WatchPrefix etcdop.PrefixT[T]
-	// WatchEvents must contain at least one of etcdop.CreateEvent, etcdop.UpdateEvent, etcdop.DeleteEvent.
-	WatchEvents []etcdop.EventType
```

Comment:
Toto tu bolo zbytocne, nema zmysel sledovat ine ako `create` events, ani to nikde nepouzivame, ...